01-main/packages/freetube

```diff
@@ -0,0 +1,9 @@
+DEFVER=1
```

Comment:
They are producing amd64, arm64 and armv7l debs
It would be good to add an appropriate ARCHS_SUPPORTED line and incorporate HOST_ARCH in the URL to make them available.